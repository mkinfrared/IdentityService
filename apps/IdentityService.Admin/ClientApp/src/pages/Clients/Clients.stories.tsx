import { Meta, Story } from "@storybook/react";

import Clients from "./Clients";
import { ClientsProps } from "./Clients.type";
import css from "./Story.module.scss";

export default {
  title: "PAGES/Clients",
  component: Clients,
  parameters: {
    componentSubtitle: "Subtitle goes here",
  },
} as Meta;

const Template: Story<ClientsProps> = (args) => (
  <div className={css.Story}>
    <Clients {...args} />
  </div>
);

const Default = Template.bind({});

Default.args = {};

Default.parameters = {
  docs: {
    storyDescription: "Story description",
  },
};

export { Default };
