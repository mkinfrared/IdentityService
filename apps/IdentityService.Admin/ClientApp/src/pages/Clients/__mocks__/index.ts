export { default } from "./Clients.mock";
