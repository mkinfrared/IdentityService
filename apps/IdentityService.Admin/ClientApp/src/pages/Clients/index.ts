export { default } from "./Clients";

export * from "./Clients.type";
