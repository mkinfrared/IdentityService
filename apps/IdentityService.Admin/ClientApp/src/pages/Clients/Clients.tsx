import css from "./Clients.module.scss";
import { ClientsProps } from "./Clients.type";

const Clients = ({ className }: ClientsProps) => {
  const classNames = [css.Clients, className].filter(Boolean).join(" ");

  return (
    <div className={classNames} data-testid="Clients">
      Clients works!!
    </div>
  );
};

export default Clients;
