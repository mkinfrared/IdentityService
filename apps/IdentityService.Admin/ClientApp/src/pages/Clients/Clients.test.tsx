import { render } from "@testing-library/react";

import Clients from "./Clients";

describe("<Clients />", () => {
  const Component = <Clients />;

  it("should be defined", () => {
    expect(Clients).toBeDefined();
  });

  it("should match the snapshot", () => {
    const { container } = render(Component);

    expect(container).toMatchSnapshot();
  });

  it("should contain a data test id", () => {
    const { getByTestId } = render(Component);
    const element = getByTestId("Clients");

    expect(element).toBeDefined();
  });
});
