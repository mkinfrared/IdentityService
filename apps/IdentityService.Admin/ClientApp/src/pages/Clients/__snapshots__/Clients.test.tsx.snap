// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Clients /> should match the snapshot 1`] = `
<div>
  <div
    class="Clients"
    data-testid="Clients"
  >
    Clients works!!
  </div>
</div>
`;
